lib/sidekiq/limit_fetch/queues.rb

```diff
@@ -3,27 +3,25 @@ module Sidekiq::LimitFetch::Queues
 
   THREAD_KEY = :acquired_queues
 
-  def start(options)
-    if Sidekiq::LimitFetch.post_7?
-      options = options.config
-    end
+  def start(capsule_or_options)
```

Comment:
This is a renaming to make it clear that the argument maybe a `Capsule` object or an options Hash, depending on the Sidekiq version